events/lastAvatarStandingTrivia/clientScripts/trivia.html

```diff
@@ -244,17 +257,14 @@ <h4>Choose Question Details (Optional) <br>& Click 'New Question'</h4>
 
             $(document).ready(function () {
                 EventBridge.scriptEventReceived.connect(onScriptEventReceived);
+                var event = {
+                    app: 'trivia',
+                    type: "listening"
+                };
+                EventBridge.emitWebEvent(JSON.stringify(event));
+                beginButton.disabled = false;
```

Comment:
do you want to enable this button _here_, or when the event from the app JS comes back?

Reply:
This enables the button in the case that a game has not been started yet.  The html now loads with all buttons disabled by default to prevent a host from starting in an error state.